status/util.py

```diff
@@ -191,7 +191,10 @@ def get(self):
                 continue
             if server not in server_status:
                 server_status[server] = row.value
-                server_status[server]['instrument'] = instruments[server] or '-'
+                try:
+                    server_status[server]['instrument'] = instruments[server] or '-'
```

Comment:
Why not use `instruments.get(server, '-')` ? The try catch wouldn't be needed then

Reply:
Good point 👍 I'll change it